test/e2e/common_test.go

```diff
@@ -130,26 +131,31 @@ var (
 )
 
 const (
-	// lockdir - do not use directly use LockTmpDir
+	// lockdir - do not use directly; use LockTmpDir
 	lockdir = "libpodlock"
 	// imageCacheDir - do not use directly use ImageCacheDir
 	imageCacheDir = "imagecachedir"
 )
 
 var _ = SynchronizedBeforeSuite(func() []byte {
-	globalTmpDir := GinkgoT().TempDir()
+	// One global scratch directory under which all test files will live.
+	// Force it under /var/tmp so as not to abuse tmpfs /tmp
+	globalTmpDir, err := os.MkdirTemp("/var/tmp", "pme2e-")
```

Comment:
It would help me but I am not sure this condition actually depends on CI or not. It depends on whenever a system has enough available RAM to run the suite fully in RAM. If I would like to run this on my small PI it most likely will not work. So maybe we should measure how much RAM we actually need and decide base don this.

Also for CI it might be worth to investigate to fully run it from memory given we want to make CI faster.